web/src/lib/data-hooks/useBusinessNameSearch.ts

```diff
@@ -41,7 +41,7 @@ export const useBusinessNameSearch = ({
     setCurrentName(isDba ? userData.profileData.nexusDbaName || "" : userData.profileData.businessName);
     if (
       userData.formationData.businessNameAvailability?.status !== "AVAILABLE" &&
-      userData.formationData.businessNameAvailability?.status !== undefined
+      userData.formationData.businessNameAvailability !== undefined
```

Comment:
It took me a bit of reading to figure out why we were checking these two values. Maybe saving as a variable would make it more readable? Something like:

```
if (businessNameHasBeenSearched && isNotAvailable)
```

Reply:
I think that's a good idea, making the change.